app/templates/people/person/positions/minister/edit.hbs

```diff
@@ -0,0 +1,260 @@
+                        @onChange={{fn
+                            (mut
+                            this.model.minister.contacts.firstObject.contactAddress.municipality
+                            )
+                        }}
+                        />
+                    </:content>
+                    </Item>
+                    <Item @labelFor="contact-address-province">
+                    <:label>Provincie</:label>
+                    <:content>
+                        <ProvinceSelect
+                        @selected={{this.model.minister.contacts.firstObject.contactAddress.province}}
+                        @onChange={{fn
+                            (mut
+                            this.model.minister.contacts.firstObject.contactAddress.province
+                            )
+                        }}
+                        />
+                    </:content>
+                    </Item>
+                </:left>
+                <:right as |Item|>
+                    <Item @labelFor="contact-email">
+                    <:label>E-mail</:label>
+                    <:content>
+                        <AuInput
+                        @width="block"
+                        @value={{this.model.minister.contacts.firstObject.email}}
+                        id="contact-email"
+                        />
+                    </:content>
+                    </Item>
+                    <Item @labelFor="contact-telephone">
+                    <:label>Telefoonnummer</:label>
+                    <:content>
+                        <AuInput
+                        @width="block"
+                        @value={{this.model.minister.contacts.firstObject.telephone}}
+                        id="contact-telephone"
+                        />
+                    </:content>
+                    </Item>
+                </:right>
+                </Card.Columns>
+            </:card>
+            </EditCard>
+
+            <EditCard>
+                <:title>Aanstelling</:title>
+                <:card as |Card|>
+                <Card.Columns>
+                    <:left as |Item|>
+                    <Item @labelFor="start-date">
+                        <:label>Startdatum</:label>
+                        <:content>
+                        <Datepicker
+                            @id="start-date"
+                            @onChange={{fn (mut this.model.minister.agentStartDate)}}
+                            @value={{this.model.minister.agentStartDate}}
+                        />
+                        </:content>
+                    </Item>
+                    </:left>
+                    <:right as |Item|>
+                    <Item @labelFor="end-date">
+                        <:label>Einddatum</:label>
+                        <:content>
+                        <Datepicker
+                            @disabled={{this.isCurrentPosition}}
```

Comment:
`this.isCurrentPosition` also needs to be determined when the page is visited. I assume it can be set to true if no `agentEndDate` is set.